pkg/rbd/nodeserver.go

```diff
@@ -553,16 +553,19 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	}
 	defer ns.VolumeLocks.Release(volumeID)
 
-	_, err := getVolumeName(volumeID)
+	volName, err := getVolumeName(volumeID)
 	if err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
-
-	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
-	devicePath, err := getDevicePath(ctx, volumePath)
+	imgInfo, devicePath, err := getDevicePathAndImageInfo(ctx, volumePath)
 	if err != nil {
 		return nil, status.Error(codes.Internal, err.Error())
 	}
+	// comapare volume name from request and volume name stored in stash
```

Comment:
Nit: "comapare" -> "compare"

It is also pretty clear what is checked, there it not really a need to explain that :-)